### ISSUE TITLE
bookmarklet fails to load jquery

### ISSUE THREAD

**Issue by thinsoldier** [browserstate/ajaxify#4]
I get endless console.log "Loading jQuery" messages.

The script element for //raw.github.com/browserstate/ajaxify/master/ajaxify-bookmarklet-helper.js gets successfully added to the end of my <body>

But the line in that file:
window.jQuery || document.write('<script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"><\/script>');

doesn't seem to actually work.

I'm using firefox 20. That document.write line doesn't seem to work when run by itself via the console. But document.write('foo'); does work.

**Comment by ghost** [comment]
Should be work fine now!

**Comment by thinsoldier** [comment]
Nope. On some sites that have no js library at all I get an infinie loop of "loading jquery" console.log calls.

On a site that already has jquery I get a blank page after clicking the bookmarklet. Seems to be some issue with document.write in firefox.

Could you add an actual anchor containing the bookmarklet code to the readme so people can just click+drag that to their bookmarks toolbar. 

I haven't tested in chrome yet because chrome's bookmarks won't let me paste the bookmarklet code into them. I probably would need to edit it to look like a real url without entities for ://

**Comment by thinsoldier** [comment]
In which browser on which platforms does the bookmarklet work for you?
